libminifi/src/FlowController.cpp

```diff
@@ -289,6 +289,12 @@ Processor *FlowControllerImpl::createProcessor(std::string name, uuid_t uuid) {
 		return NULL;
 	}
 
+	#ifdef BUILD_GPS
```

Comment:
This was not very smart of me ... this will cause an error to be thrown that the processor of type GetGPS does not exist at runtime because the last else condition will be triggered before this. I need to move this higher up the condition before the last else.